CHANGELOG.md

```diff
@@ -4,10 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+<!-- Template: after a release, copy and paste out below
```

Comment:
You can remove this.

Reply:
Oh, you don't like my wonderful invention? 

This does not render in HTML. 
When starting a new dev cycle, it saves one the task of creating fresh headings for the log sections, either by typing them in (risking typos) or by copy-pasting each individual line.  Now one can cut-and-paste the whole block and be done with  it.

Reply:
Or did you mean to merely remove the text `Template: .... below`?